docs/development/workflow.rst

```diff
@@ -0,0 +1,6 @@
+=============================
+How to contribute to PlasmaPy
+=============================
+
+Many ways to contribute
+=======================
```

Comment:
I suppose we'll expand this part in a follow-up pull request right? 😉

Reply:
Yep!  We'll be working on that soon! 🐱

Reply:
😸